argparse_dataclass.py

```diff
@@ -298,8 +298,6 @@ def format_usage(self):
 # In Python 3.10, we can use types.NoneType
 NoneType = type(None)
 
-__version__ = "2.0.1"
```

Comment:
I would like to keep `__version__` defined. One way to do this could be something like [this](https://stackoverflow.com/a/67097076/6628184).

Reply:
Good idea. I ended up defining the `__version__` parameter as suggested in the stackoverflow post you linked.